Rules/mtbmap.mrules

```diff
@@ -1792,49 +1807,46 @@ rules
 				fill-hatch-color : #53B4EC
 				fill-hatch-opacity : 0.1
 				painting-cycle : 450
-				text-color : #3290CB
-			draw : fill
-			define
-				min-zoom : 15
-			draw : text
-			
+
+
 		if : * lawn
 			define
 				line-style : none
 				min-zoom : 12
 				fill-color : #DEFED2
 				painting-cycle : 403
-			draw : fill
-			
+
+		
 		if : * orchard
 			define
 				line-style : none
 				fill-texture : Textures/orchard.png
 				painting-cycle : 402
 				min-zoom : 12
-			draw : fill
-			
+
+				
 		if : * scrub
 			define
 				line-style : none
 				min-zoom : 10
 				fill-texture : Textures/scrub.png
 				painting-cycle : 402
-			draw : fill
-			
+
+				
 		if : * sand
 			define
+				text-func : NodeName(e)
+				min-zoom : 13
+			for : not natural=beach
+				draw : text
```

Comment:
I'm not sure that this `draw : text` will be affected by the following `text-func : NodeName(e)`

This will be fixed by having a single `text-func : NodeName(e)` for `area *`